common/djangoapps/enrollment/data.py

```diff
@@ -75,7 +75,7 @@ def get_course_enrollment(username, course_id):
         A serializable dictionary representing the course enrollment.
 
     """
-    course_key = CourseKey.from_string(course_id)
```

Comment:
is this an appropriate place to call this helper? is this function only called by a view?